source/binding/Bitstream.cpp

```diff
@@ -178,15 +202,33 @@ static size_t bitstreamCastRemainingSize(const T& destination, size_t srcSize) {
     // Calculate remaining size to dynamically fill.
     auto [destFillMultiplier, destFillFixedSize] =
         dynamicBitstreamSize(destination, BitstreamSizeMode::DestFill);
-    if (srcSize < destFillFixedSize || (srcSize - destFillFixedSize) % destFillMultiplier != 0)
+    if (srcSize < destFillFixedSize || (srcSize - destFillFixedSize) % destFillMultiplier != 0) {
+        if (destEmptyMultiplier > 0 && (srcSize - destEmptyFixedSize) % destEmptyMultiplier == 0)
+            return 0; // only for "with" range
         return srcSize + 1;
+    }
 
     // Size to fill the first dynamically sized item.
     return srcSize - destFillFixedSize;
 }
 
+/// Formats the width of a dynamically-sized bitstream to a formula like 8×n+3 in an error message
+template<typename T>
+static std::string formatWidth(const T& bitstream, bool destination = false) {
+    FormatBuffer buffer;
+    auto mode = destination ? BitstreamSizeMode::DestFill : BitstreamSizeMode::Source;
+    auto [multiplier, fixedSize] = dynamicBitstreamSize(bitstream, mode);
+    if (!multiplier)
+        buffer.format("{}", fixedSize);
+    else if (!fixedSize)
+        buffer.format("{}×n", multiplier);
```

Comment:
Should probably just use '*' instead of the special character.

Reply:
Yes, I replace the fancy unicode multiplication symbol with ASCII character '*'.